apps/react-18-tests-v9/package.json

```diff
@@ -19,14 +19,17 @@
     "@fluentui/scripts-tasks": "*"
   },
   "dependencies": {
+    "@cypress/react": "9.0.0",
```

Comment:
isn't this for react v 19?

Reply:
They actually work for both versions (18 and 19). The main "cypress" is the one that don't work currently for the version we have installed.